### PR TITLE
Integrate auth-client for Hydra logout functionality

### PR DESCRIPTION
**Pull request by adrienne-deriv**
Added Hydra authentication feature for `deriv-api-docs` under the feature flag `hydra_be`, this feature will allow SSO to be integrated to `api.deriv.com`

### CODE REVIEW

**Review by thisyahlen-deriv — APPROVED** [review]

**Review by thisyahlen-deriv — APPROVED** [review]

**Review by thisyahlen-deriv — APPROVED** [review]
LGTM

**Review by thisyahlen-deriv — APPROVED** [review]

File: src/components/AccountSwitcher/index.tsx [hunk 1]
```diff
@@ -10,8 +12,15 @@ import styles from './account_switcher.module.scss';
 import SearchButton from '../SearchButton';
 import Translate from '@docusaurus/Translate';
 
+
```
Comment (thisyahlen-deriv) [inline src/components/AccountSwitcher/index.tsx:15]: remove space

**Review by thisyahlen-deriv — APPROVED** [review]

File: .github/actions/build/action.yml [hunk 2]
```diff
@@ -14,9 +14,6 @@ inputs:
   GROWTHBOOK_CLIENT_KEY:
     description: Growthbook client key for analytics
     required: false
-  GROWTHBOOK_DECRYPTION_KEY:
-    description: Growthbook decryption key for analytics
-    required: false
```
Comment (adrienne-deriv) [inline .github/actions/build/action.yml:19]: GROWTHBOOK_DECRYPTION_KEY is not needed in the new analytics package update, thus removing it

**Review by sandeep-deriv — APPROVED** [review]

File: src/hooks/useGrowthbookGetFeatureValue/index.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react';
 import { Analytics } from '@deriv-com/analytics';
-import useIsGrowthbookIsLoaded from './useIsGrowthbookLoaded';
+import useIsGrowthbookIsLoaded from '../useIsGrowthbookLoaded';
```
Comment (amam-deriv) [inline src/hooks/useGrowthbookGetFeatureValue/index.tsx:3]: is this changes intended?

**Review by amam-deriv — APPROVED** [review]

File: src/hooks/useGrowthbookGetFeatureValue/index.tsx (hunk 3 above)
Comment (adrienne-deriv) [reply src/hooks/useGrowthbookGetFeatureValue/index.tsx:3]: Yes, previously it was a file in `src/hooks/useGrowthbookGetFeatureValue.tsx`, moved it to a folder for consistency with other hooks

**Review by prince-deriv — APPROVED** [review]
